db/collection_test.go

```diff
@@ -20,12 +21,14 @@ const (
 
 type Person struct {
 	ID   core.InstanceID `json:"_id"`
+	Mod  int64           `json:"_mod"`
```

Comment:
Added for tests, this _isn't_ needed by end-users. They can leave this off and things will work just fine.